.eslintrc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   'extends': 'airbnb',
   'rules': {
     'eqeqeq': ['error', 'allow-null'],
+    'import/no-unresolved': 'off',
```

Comment:
Could you include a link to a bug report as a comment, so we know when to re-enable this, as it is quite a handy rule for catching typos.